lms/static/coffee/src/instructor_dashboard/membership.coffee

```diff
@@ -374,12 +376,18 @@ class BatchEnrollment
 
     # attach click handler for enrollment buttons
     @$enrollment_button.click (event) =>
+      if @$is_course_white_label
+        if not @$reason_field.val()
+          @fail_with_error gettext "Reason field should not be left blank."
+          return false
+
```

Comment:
Hrm. It seems like you are properly checking if it is a white label course here. But looking at the <input type='hidden' ..> in the HTML, you are just putting the Python boolean which I think will serialize to 'True' and 'False', both of which will always be treated as a true statement in JavaScript. Here if you have to explicitly check to see:

if @$is_course_white_label == 'True'